perception/detection_class_adapter/config/detection_class_adapter.param.yaml

```diff
@@ -0,0 +1,37 @@
+/**:
+  ros__parameters:
+    allow_remapping_by_area_matrix:
+      # NOTE(kl): We turn all vehicles into trailers if they go over 3x12 m2.
+      # NOTE(kl): We turn cars into trucks if they have an area between 2.2 x 5.5 and 2.5 * 7.9 m2
+      #UNKNOWN, CAR, TRUCK, BUS,  TRAILER, MOTORBIKE, BICYCLE,PEDESTRIAN <-Measurement
+      [0,       0,   0,     0,    0,       0,         0,      0,         #UNKNOWN <-Tracker
```

Comment:
Is the comment a mistake about Tracker?

Reply:
Yes, I initially copied the tracker's param file to use as a template. I will fix it together with hard-coded class remapping.